transcript-client/src/hooks/useEditor.ts

```diff
@@ -100,7 +100,7 @@ function useEditorHook({
         setEditorState(EditorState.forceSelection(newEditorState, selection));
       }
     }
-  }, [ allHighlightColors]);
+  }, [allHighlightColors, editorState, setEditorState]);
```

Comment:
Passing editorState and setEditorState as a dependency in the dependency array causes an issue. That why it wasn't included at first.

Reply:
Omitting editorState and setEditorState caused the react build script to throw an error.